# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/uucp/uucp-1.06.1-misc.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- uucp-1.06.1.orig/cu.1	Sun Aug 20 13:29:24 1995
+++ uucp-1.06.1/cu.1	Fri Apr 18 11:34:08 1997
@@ -304,7 +304,9 @@
 approximation.
 
 .br
-/usr/lib/uucp/config - Configuration file.
+/etc/uucp/config - Configuration file.
+.br
+/etc/uucp/oldconfig/ - Alternative configuration (HDB or V2 format)
 .SH AUTHOR
 Ian Lance Taylor
 <ian@airs.com>
diff -uNr uucp-1.06.1.orig/policy.h uucp-1.06.1/policy.h
--- uucp-1.06.1.orig/policy.h	Sun Aug 20 13:36:06 1995
+++ uucp-1.06.1/policy.h	Fri Apr 18 11:34:08 1997
@@ -241,7 +241,7 @@
    the sendmail choice below.  Otherwise, select one of the other
    choices as appropriate.  */
 #if 1
-#define MAIL_PROGRAM "/usr/lib/sendmail -t"
+#define MAIL_PROGRAM "/usr/sbin/sendmail -t"
 #define MAIL_PROGRAM_TO_BODY 1
 #define MAIL_PROGRAM_SUBJECT_BODY 1
 #endif
@@ -302,6 +302,7 @@
 /* #define LOCKDIR "/usr/spool/locks" */
 /* #define LOCKDIR "/usr/spool/uucp/LCK" */
 /* #define LOCKDIR "/var/spool/lock" */
+#define LOCKDIR "/var/lock"
 
 /* You must also specify the format of the lock files by setting
    exactly one of the following macros to 1.  Check an existing lock
@@ -465,8 +466,8 @@
    When looking something up (a system, a port, etc.) the new style
    configuration files will be read first, followed by the V2
    configuration files, followed by the HDB configuration files.  */
-#define HAVE_V2_CONFIG 0
-#define HAVE_HDB_CONFIG 0
+#define HAVE_V2_CONFIG 1
+#define HAVE_HDB_CONFIG 1
 
 /* Exactly one of the following macros must be set to 1.  The exact
    format of the spool directories is explained in unix/spool.c.
@@ -571,7 +572,7 @@
 /* The name of the default spool directory.  If HAVE_TAYLOR_CONFIG is
    set to 1, this may be overridden by the ``spool'' command in the
    configuration file.  */
-#define SPOOLDIR "/usr/spool/uucp"
+#define SPOOLDIR "/var/spool/uucp"
 /* #define SPOOLDIR "/var/spool/uucp" */
 
 /* The name of the default public directory.  If HAVE_TAYLOR_CONFIG is
@@ -579,7 +580,7 @@
    configuration file.  Also, a particular system may be given a
    specific public directory by using the ``pubdir'' command in the
    system file.  */
-#define PUBDIR "/usr/spool/uucppublic"
+#define PUBDIR "/var/spool/uucppublic"
 /* #define PUBDIR "/var/spool/uucppublic" */
 
 /* The default command path.  This is a space separated list of
@@ -643,19 +644,19 @@
 /* The default log file when using HAVE_TAYLOR_LOGGING.  When using
    HAVE_TAYLOR_CONFIG, this may be overridden by the ``logfile''
    command in the configuration file.  */
-#define LOGFILE "/usr/spool/uucp/Log"
+#define LOGFILE "/var/log/uucp/Log"
 /* #define LOGFILE "/var/spool/uucp/Log" */
 
 /* The default statistics file when using HAVE_TAYLOR_LOGGING.  When
    using HAVE_TAYLOR_CONFIG, this may be overridden by the
    ``statfile'' command in the configuration file.  */
-#define STATFILE "/usr/spool/uucp/Stats"
+#define STATFILE "/var/log/uucp/Stats"
 /* #define STATFILE "/var/spool/uucp/Stats" */
 
 /* The default debugging file when using HAVE_TAYLOR_LOGGING.  When
    using HAVE_TAYLOR_CONFIG, this may be overridden by the
    ``debugfile'' command in the configuration file.  */
-#define DEBUGFILE "/usr/spool/uucp/Debug"
+#define DEBUGFILE "/var/log/uucp/Debug"
 /* #define DEBUGFILE "/var/spool/uucp/Debug" */
 
 #endif /* HAVE_TAYLOR_LOGGING */
diff -uNr uucp-1.06.1.orig/uucico.8 uucp-1.06.1/uucico.8
--- uucp-1.06.1.orig/uucico.8	Sun Aug 20 13:29:42 1995
+++ uucp-1.06.1/uucico.8	Fri Apr 18 11:35:54 1997
@@ -250,8 +250,8 @@
 .B \-\-debug abnormal,chat.
 
 The debugging output is sent to the debugging file, usually one of
-/usr/spool/uucp/Debug, /usr/spool/uucp/DEBUG, or
-/usr/spool/uucp/.Admin/audit.local.
+/var/log/uucp/Debug, /usr/spool/uucp/DEBUG, or
+/var/spool/uucp/.Admin/audit.local.
 .TP 5
 .B \-I file, \-\-config file
 Set configuration file to use.  This option may not be available,
@@ -269,20 +269,22 @@
 configuration file, so these are only approximations.
 
 .br
-/usr/lib/uucp/config - Configuration file.
+/etc/uucp/config - Configuration file
 .br
-/usr/lib/uucp/passwd - Default UUCP password file.
+/etc/uucp/oldconfig/ - Alternative configuration (HDB or V2 format)
 .br
-/usr/spool/uucp -
+/etc/uucp/passwd - Default UUCP password file.
+.br
+/var/spool/uucp -
 UUCP spool directory.
 .br
-/usr/spool/uucp/Log -
+/var/log/uucp/Log -
 UUCP log file.
 .br
-/usr/spool/uucppublic -
+/var/spool/uucppublic -
 Default UUCP public directory.
 .br
-/usr/spool/uucp/Debug -
+/var/log/uucp/Debug -
 Debugging file.
 .SH SEE ALSO
 kill(1), uucp(1), uux(1), uustat(1), uuxqt(8)
diff -uNr uucp-1.06.1.orig/uucp.1 uucp-1.06.1/uucp.1
--- uucp-1.06.1.orig/uucp.1	Sun Aug 20 13:29:46 1995
+++ uucp-1.06.1/uucp.1	Fri Apr 18 11:36:22 1997
@@ -181,15 +181,17 @@
 configuration file, so these are only approximations.
 
 .br
-/usr/lib/uucp/config - Configuration file.
+/etc/uucp/config - Configuration file.
 .br
-/usr/spool/uucp -
+/etc/uucp/oldconfig/ - Alternative configuration (HDB or V2 format)
+.br
+/var/spool/uucp -
 UUCP spool directory.
 .br
-/usr/spool/uucp/Log -
+/var/log/uucp/Log -
 UUCP log file.
 .br
-/usr/spool/uucppublic -
+/var/spool/uucppublic -
 Default UUCP public directory.
 .SH SEE ALSO
 mail(1), uux(1), uustat(1), uucico(8)
diff -uNr uucp-1.06.1.orig/uustat.1 uucp-1.06.1/uustat.1
--- uucp-1.06.1.orig/uustat.1	Sun Aug 20 13:29:55 1995
+++ uucp-1.06.1/uustat.1	Fri Apr 18 11:34:08 1997
@@ -540,9 +540,11 @@
 configuration file, so these are only approximations.
 
 .br
-/usr/lib/uucp/config - Configuration file.
+/etc/uucp/config - Configuration file.
 .br
-/usr/spool/uucp -
+/etc/uucp/oldconfig/ - Alternative configuration (HDB or V2 format)
+.br
+/var/spool/uucp -
 UUCP spool directory.
 .SH SEE ALSO
 ps(1), rmail(1), uucp(1), uux(1), uucico(8), uuxqt(8)
diff -uNr uucp-1.06.1.orig/uux.1 uucp-1.06.1/uux.1
--- uucp-1.06.1.orig/uux.1	Sun Aug 20 13:29:57 1995
+++ uucp-1.06.1/uux.1	Fri Apr 18 11:35:17 1997
@@ -40,7 +40,7 @@
 not absolute, it will be appended to the current working directory on
 the local system; the result may not be meaningful on the remote
 system.  A pathname may begin with ~/, in which case it is relative to
-the UUCP public directory (usually /usr/spool/uucppublic) on the
+the UUCP public directory (usually /var/spool/uucppublic) on the
 appropriate system.  A pathname may begin with ~name/, in which case
 it is relative to the home directory of the named user on the
 appropriate system.
@@ -231,15 +231,17 @@
 configuration file, so these are only approximations.
 
 .br
-/usr/lib/uucp/config - Configuration file.
+/etc/uucp/config - Configuration file.
 .br
-/usr/spool/uucp -
+/etc/uucp/oldconfig/ - Alternative configuration (HDB or V2 format)
+.br
+/var/spool/uucp -
 UUCP spool directory.
 .br
-/usr/spool/uucp/Log -
+/var/log/uucp/Log -
 UUCP log file.
 .br
-/usr/spool/uucppublic -
+/var/spool/uucppublic -
 Default UUCP public directory.
 .SH SEE ALSO
 mail(1), uustat(1), uucp(1), uucico(8), uuxqt(8)
diff -uNr uucp-1.06.1.orig/uuxqt.8 uucp-1.06.1/uuxqt.8
--- uucp-1.06.1.orig/uuxqt.8	Sun Aug 20 13:29:58 1995
+++ uucp-1.06.1/uuxqt.8	Fri Apr 18 11:36:13 1997
@@ -64,8 +64,8 @@
 .B \-\-debug abnormal,chat.
 
 The debugging output is sent to the debugging file, usually one of
-/usr/spool/uucp/Debug, /usr/spool/uucp/DEBUG, or
-/usr/spool/uucp/.Admin/audit.local.
+/var/log/uucp/Debug, /usr/spool/uucp/DEBUG, or
+/var/spool/uucp/.Admin/audit.local.
 .TP 5
 .B \-I file, \-\-config 
 Set configuration file to use.  This option may not be available,
@@ -83,18 +83,20 @@
 configuration file, so these are only approximations.
 
 .br
-/usr/lib/uucp/config - Configuration file.
+/etc/uucp/config - Configuration file.
 .br
-/usr/spool/uucp -
+/etc/uucp/oldconfig/ - Alternative configuration (HDB or V2 format)
+.br
+/var/spool/uucp -
 UUCP spool directory.
 .br
-/usr/spool/uucp/Log -
+/var/log/uucp/Log -
 UUCP log file.
 .br
-/usr/spool/uucppublic -
+/var/spool/uucppublic -
 Default UUCP public directory.
 .br
-/usr/spool/uucp/Debug -
+/var/log/uucp/Debug -
 Debugging file.
 .SH SEE ALSO
 uucp(1), uux(1), uucico(8)
